### PR TITLE
Blog/understanding sql dialects: How Different Databases Speak Their Own SQL Language

### PR DESCRIPTION
**Pull request by hulksyed07**
## What did you do?

Please include a summary of the changes.

- [x] Added this
- [ ] Updated that

## Why did you do it?

Why were these changes made?

- This was missing

## Screenshots (Please include if anything visual)

Include any relevant screenshots that may help explain the change.

### CODE REVIEW

File: content/blog/understanding-sql-dialects.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
++++
+title = "Understanding SQL Dialects: How Different Databases Speak Their Own SQL Language"
+slug = "demo-post"
+date = 2024-06-26T11:19:22+05:30
+image = "/images/2024/understanding_sql_dialects.png"
+draft = true
+authors = ["Syed Mohd Mehndi"]
+description = "This blog post aims to provide a general overview of how different DBMSs extend standard SQL with their unique features, making it easier for developers to understand and utilize these systems effectively."
+tags = ["SQL", "Database", "RDBMS", "Postgres", "Oracle", "SQL Server"]
+categories = ["SQL", "Database"]
```
Comment (2KAbhishek) [inline content/blog/understanding-sql-dialects.md:10]: - Lets update the slug to something like `understanding-sql-dialects`
- Before merging, draft needs to be set as `false`
- @swathynair19 can you please suggest some existing tags/categories that this blog can be linked to

Comment (swathynair19) [reply content/blog/understanding-sql-dialects.md]: We can keep the tags short to just two:  "SQL" and "DBMS".

Comment (hulksyed07) [reply content/blog/understanding-sql-dialects.md]: Thanks @2KAbhishek  resolved point number 1,2 and 3
@swathynair19 updated the file to use only "SQL" and "RDBMS"

**Review by 2KAbhishek — APPROVED** [review]
Content looks good and informative, I did miss reading about MySQL, if you are aware of any features specific to MySQL, it'll be a good addition here :)

**Comment by hulksyed07** [comment]
> Content looks good and informative, I did miss reading about MySQL, if you are aware of any features specific to MySQL, it'll be a good addition here :)

Thanks Abhishek, Good point
Let me add mysql section too

**Comment by hulksyed07** [comment]
> Content looks good and informative, I did miss reading about MySQL, if you are aware of any features specific to MySQL, it'll be a good addition here :)

Done, added mysql information

**Pull request merged**